src/test/java/org/example/AppTest.java

```diff
@@ -0,0 +1,42 @@
+package org.example;
+
+import junit.framework.TestCase;
+
+
+import java.sql.*;
+
+public class AppTest extends TestCase {
+    public static final String JDBC_DRIVER = "org.postgresql.Driver";
+    static final String DB_URL = "jdbc:postgresql://localhost:5432/mydb";
+    static final String USER = "postgres";
+    static final String PASS = "root";
+    public void testApp() throws ClassNotFoundException {
+        Connection conn;
+        Statement stmt;
+        try {
```

Comment:
unit tests are not allowed to query a live database, remind me please to discuss this in class